content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -76,6 +76,10 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cl
 EOF
 yum install -y kubectl
 {{< /tab >}}
+{{< tab name="OpenSUSE Tumbleweed or Leap 15.0" codelang="bash" >}}
```

Comment:
“Leap 15.0” seems like a specific OS release. Is that right?

Reply:
Correct. There's no official package for kubectl in the latest release of Leap (15.1, see https://software.opensuse.org/package/kubectl). There's a community alternative but I wasn't sure whether it's appropriate to include it here.

If the community package was used, the changes would look something like this:

```
{{< tab name="OpenSUSE" codelang="bash" >}}
# Extra step for Leap 15.1
sudo zypper ar -e -f https://download.opensuse.org/repositories/home:/so_it_team/openSUSE_Leap_15.1/home:so_it_team.repo

# All distributions
sudo zypper refresh
sudo zypper in -y kubectl
{{< /tab >}}
```

Reply:
I think @sftim was just concerned that putting in a specific OS release would quickly cause the docs to become out of date. Though I will note that the Ubuntu/Debian tab is hardcoding "kubernetes-xenial main" in the deb setup, and the RHEL/Centos tab is hardcoding "kubernetes-el7-x86_64" while keeping the tab title free of any specific OS version.

Perhaps best to hardcode a similar package repository for the OpenSUSE tab contents? So, something like this?

```
{{< tab name="OpenSUSE" codelang="bash" >}}
sudo zypper ar -e -f https://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Tumbleweed/Virtualization:containers.repo
sudo zypper refresh
sudo zypper in -y kubectl
{{< /tab >}}
```